pages/api/get-youtube-playlist.ts

```diff
@@ -0,0 +1,42 @@
+import { NextApiRequest, NextApiResponse } from "next";
+
+import { cache } from "../../services/server/cacheService";
+const CACHE_HOURS = 6 * 60 * 60; // ~ 6 hours
+
+import { getYoutubePlaylist } from "../../services/server/youtube";
+
+type NextApiRequestWithParams = NextApiRequest & {
+  query: {
+    playlistId: string;
+    videosCount: number;
+  };
+};
+
+export default async function handler(
+  req: NextApiRequestWithParams,
+  res: NextApiResponse
+) {
+  if (req.method === "GET") {
```

Comment:
I'd prefer to invert this `if` and fast return to reduce cyclomatic complexity

Reply:
Agree, Fixed it